docs/ConsumerExample.md

```diff
@@ -64,6 +64,80 @@ signalTraps.map(type => {
 })
 ```
 
+## <a name="typescript-consumer-example"></a> TypeScript Example
+
+A similar example in TypeScript
+
+```typescript
+import { Consumer, ConsumerSubscribeTopic, EachBatchPayload, Kafka, EachMessagePayload } from 'kafkajs'
+
+// ExampleMessageProcessor is a placeholder for the logic applied to each 
```

Comment:
I think it's better if we remove this, to reduce the risk of confusion. Let's just have the eachMessage handler console.log the message value, like we do in the JS example.

Reply:
Updated.